### PR TITLE
Fix #141: Remove old `extension on` syntax

### LINKED ISSUE #141: Extension token is incorrectly highlighted as keyword instead of identifier
The `extension` token is incorrectly highlighted as a keyword in the screenshot below

Screenshot 2020-09-04 at 10 14 05

I expected `extension` to be highlighted as an identifier. cc/ @nicolasstucki

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: tests/unit/#141.test.scala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// SYNTAX TEST "source.scala"
+
+val language = file.extension {
+//                  ^^^^^^^^^ - keyword.declaration.scala
+val language = file extension {
+//                  ^^^^^^^^^ - keyword.declaration.scala
+val language = file.extension(
+//                  ^^^^^^^^^ - keyword.declaration.scala
+val language = file extension (
+//                  ^^^^^^^^^ - keyword.declaration.scala
```
Comment (olafurpg) [inline tests/unit/#141.test.scala:10]: Does this assert its highlighted as a keyword? I expected it to be highlighted the same as “file”

**Review by olafurpg — COMMENTED** [review]
Thank you for looking into this!

File: tests/unit/#141.test.scala (hunk 1 above)
Comment (nicolasstucki) [reply tests/unit/#141.test.scala:10]: The `-` checks that it is not a `keyword.declaration.scala`

**Review by olafurpg — APPROVED** [review]

File: tests/unit/#141.test.scala (hunk 1 above)
Comment (olafurpg) [reply tests/unit/#141.test.scala:10]: Aha, I'm not so familiar with the testing infrastructure. Great!

Comment (nicolasstucki) [reply tests/unit/#141.test.scala:10]: I discovered that one today. It must be new, I don't remember seeing it in the first version of the testing infrastructure we use.

**Pull request merged**